packages/engine/lib/stateful/src/field/key.rs

```diff
@@ -11,6 +11,11 @@ use crate::{
 pub struct RootFieldKey(String);
 
 impl RootFieldKey {
+    /// Creates a key from a [`String`].
```

Comment:
`RootFieldKey` instead of key?

Reply:
Changed in ab54572227b46bdb1d910cb84bbc775b07cdbd0d